src/picosvg/svg_types.py

```diff
@@ -239,8 +255,24 @@ def round_floats(self, ndigits: int, inplace=False) -> "SVGShape":
         return target
 
     def almost_equals(self, other: "SVGShape", tolerance: int) -> bool:
```

Comment:
what we call `tolerance` here is actually the `ndigits` decimal places for the `round_floats` method. It makes sense to keep `ndigits` for `round_floats`, but less so for an `almost_equals` method. I would keep the current name of the parameter `tolerance` (it's an "absolute" tolerance, really) and as the type instead of an `int` accept a `float`; this would normally be some negative power of 10.

If we do that, then makes sense to also change all the other `tolerance: int` parameters (in `normalize` and `affine_between` public function) to also take a `float` representing the absolute tolerance between two numbers.

Reply:
What's the advantage of passing float 10^-N rather than passing in N?

Reply:
i think it conveys better the idea of a "tolerance" as in the "(absolute) difference between two floats". One may want to accept two floats as similar if their absolute difference is within some number which may not necessarily be a multiple of 10**-n

Reply:
Not use 10^-something?! Who would do that?! Fair point, I think it makes sense for caller to provide a float tolerance then, and perhaps nanoemoji will pick its tolerance based on what it knows about the coordinate system.